src/coreclr/vm/codeman.cpp

```diff
@@ -1525,7 +1525,16 @@ void EEJitManager::SetCpuInfo()
 
     if (((cpuFeatures & ARM64IntrinsicConstants_Sve) != 0) && CLRConfig::GetConfigValue(CLRConfig::EXTERNAL_EnableArm64Sve))
     {
-        CPUCompileFlags.Set(InstructionSet_Sve);
+        int maxVectorLength = (maxVectorTBitWidth >> 3);
+        uint64_t systemVectorTLength = GetSystemVectorLength();
+
+        if (maxVectorLength >= systemVectorTLength)
+        {
+            // Enable SVE only when user specified vector length larger than or equal to the system
+            // vector length. When eabled, SVE would use full vector length available to the process.
+            // For a 256-bit machine, if user provides DOTNET_MaxVectorTBitWidth=128, disable SVE.
```

Comment:
This comment is implied by the previous statement. Is it just giving an example? If so, can we explicitly state so, such as:
> For example, if the reported SVE length is 256-bits but the `MaxVectorTBitWidth=128`, then we need to disable SVE support.

Reply:
Put a bit more concise comment now.